src/analysis/normed_space/finite_dimension.lean

```diff
@@ -621,9 +621,10 @@ begin
 end
 
 variable (𝕜)
-/-- Riesz's theorem: if the unit ball is compact in a vector space, then the space is
-finite-dimensional. -/
-theorem finite_dimensional_of_is_compact_closed_ball {r : ℝ} (rpos : 0 < r)
+
+/-- Riesz's theorem: if a closed ball of positive radius is compact in a vector space, then the
```

Comment:
```suggestion
/-- **Riesz's theorem**: if a closed ball of positive radius is compact in a vector space, then the
```

Reply:
Also add that the center is zero in this docstring? And make it bold also in the next docstring?